### ISSUE TITLE
GetThreadInfo doesn't work on group chats

### ISSUE THREAD

**Issue by DanMossa** [carpedm20/fbchat#76]
GetThreadInfo does not work in group chats.
The error I'm having is

```

Traceback (most recent call last):
  File "/opt/pycharm-com-eap/helpers/pydev/pydevd.py", line 1581, in <module>
    globals = debugger.run(setup['file'], None, None, is_module)
  File "/opt/pycharm-com-eap/helpers/pydev/pydevd.py", line 964, in run
    pydev_imports.execfile(file, globals, locals)  # execute the script
  File "/home/daniel/Dropbox/Code/Python/TesFBChat.py", line 97, in <module>
    lastMessageFromThread = (client.getThreadInfo(threadFbId, 0, 0)[0].body).upper()
  File "/home/daniel/.local/lib/python2.7/site-packages/fbchat/client.py", line 354, in getThreadInfo
    for message in j['payload']['actions']:
KeyError: 'actions'
```

**Comment by carpedm20** [comment]
Maybe solved with #89

**Comment by madsmtm** [comment]
Fixed in v. `1.0.x`
